### DISCUSSION TITLE
Filter Kinds in the sidebar

### DISCUSSION THREAD

**Discussion by josegonzalez** [getseabird/seabird#100] [Ideas]
### Summary

Add a filter in the sidebar for different kinds.

### Motivation

If you use CRDs, there could be hundreds/thousands of Kinds shown in the sidebar, making filtering annoying.

### Proposal

Just add a filter to the top of the sidebar, same as how there is one for normal resources.

**Comment by jgillich** [comment]
Coming soon
[Screencast from 2024-07-10 01-16-43.webm](https://github.com/getseabird/seabird/assets/347965/10dcaba2-36eb-43d3-8700-ded39ffa2512)
